Cargo.toml

```diff
@@ -11,4 +11,11 @@ keywords = ["jwt", "web", "api", "token", "json"]
 
 [dependencies]
 rustc-serialize = "^0.3"
-ring = "^0.7"
+error-chain = "0.10"
```

Comment:
I'd pretty much love for this to be `default-features = false`, as it opts-out of a -sys dependency in an otherwise native library.